### DISCUSSION TITLE
getBricks() PHP Function Not Working in Pimcore project

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15091] [Q&A / Support]
Hi , 

$product->getBricks() -Function Not Working

Unfortunately, it seems that this function is not working as expected. When I call getBricks() on a Data Object that contains Object brick fields, I am not able to retrieve the Object brick data.

I have tried both retrieving and setting data using the getBricks() function and corresponding setters, but neither seem to be functioning properly.

I have checked my code and verified that the Objectbrick data is being saved correctly to the database. However, when I attempt to retrieve the data using getBricks(), I receive an empty object.

This issue is causing significant delays in the development of my project, and I would greatly appreciate any assistance you can provide in resolving it.

**Comment by arulselvan008** [comment]
anybody Can you help us here?

**Reply by graceman9** [reply]
@arulselvan008  Have you found a solution?
